dist.ini

```diff
@@ -20,11 +20,7 @@ inherit_version = 0    ;optional flag
 [License]
 [ExtraTests]
 [PodSyntaxTests]
-[PodWeaver]
```

Comment:
Looks like this was being used to build the README above. Looking at why this was breaking for you in 5.26.